Resources/Prototypes/Entities/Mobs/Species/diona.yml

```diff
@@ -35,6 +35,44 @@
         amount: 5
   - type: Bloodstream
     bloodReagent: Water
+  - type: Reactive
+    groups:
+      Flammable: [ Touch ]
+      Extinguish: [ Touch ]
+      Acidic: [Touch, Ingestion]
+    reactions:
+    - reagents: [Water, SpaceCleaner]
+      methods: [Touch]
+      effects:
+      - !type:WashCreamPieReaction
+    - reagents: [ PlantBGone ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        scaled: true
+        damage:
+          types:
+            Blunt: 7
+            Slash: 5
+            Piercing: 8
+      - !type:PopupMessage
+        type: Local
+        visualType: Large
+        messages: [ "diona-hurt-by-herbicide-popup" ]
+        probability: 0.66
+    - reagents: [ WeedKiller ]
+      methods: [ Touch ]
+      effects:
+      - !type:HealthChange
+        scaled: true
+        damage:
+          types:
+            Poison: 11
```

Comment:
Ditto for this, poison is tough to heal and capable of perma-killing very easily so water guns should not be doing this much.

Reply:
Dropped to 5 Poison (20 shot kill, 40 shot round remover).